netbsd/Platform.c

```diff
@@ -347,13 +347,18 @@ bool Platform_getDiskIO(DiskIOData* data) {
 
       iostats = xRealloc(iostats, size);
 
+      errno = 0;
+
       if (sysctl(mib, __arraycount(mib), iostats, &size, NULL, 0) == 0)
          break;
 
       if (errno != ENOMEM)
          CRT_fatalError("Unable to get disk IO statistics");
    }
 
+   if (errno == ENOMEM)
+      CRT_fatalError("Unable to get disk IO statistics");
```

Comment:
Please use a different error message than used 4 lines above, to distinguish them.